amazon_transcribe/deserialize.py

```diff
@@ -92,13 +96,23 @@ def parse_start_stream_transcription_response(
         session_id = headers.get("x-amzn-transcribe-session-id")
         vocab_filter_name = headers.get("x-amzn-transcribe-vocabulary-filter-name")
         vocab_filter_method = headers.get("x-amzn-transcribe-vocabulary-filter-method")
+        show_speaker_label = ensure_boolean(
```

Comment:
One thing to note here is that if the `get` returns `None` (as in the service does not send it over the wire) we would end up populating `False` here. I'm not 100% sure if that's the behavior we'd want here.

Reply:
I'm torn on it too. I don't think `None` provides value here which is where the suggestion to default to `False` came from. If there's a compelling reason to denote an explicit False and that the server didn't send this header, then I could sway.